wiki/Storyboard/Scripting/Variables/en.md

```diff
@@ -1,22 +1,22 @@
 # Storyboard scripting variables
 
-You can strings of text, up to a whole line, as **variables** for use elsewhere in an .osb file (note that the value can't change during the gameplay, so for those with software experience, think of it more like a constant). *.osu files do not support this.*
+**Variables** are abstract containers holding custom values that can be used elsewhere in an `.osb` file (note that the value cannot be changed during gameplay, so for those with programming experience, think of it more like a constant). `.osu` files *do not* support this.
```

Comment:
Tell me if I should remove the part in brackets too. I'm not sure if this is too informal.

Reply:
```suggestion
**Variables** are custom aliases for other values, typically long or common strings, which can be reused elsewhere in `.osb` files. These cannot be changed dynamically during gameplay, meaning they are constant values. They are *not supported* in `.osu` files.
```